CONTRIBUTING.md

```diff
@@ -21,6 +21,22 @@ To install in development mode, follow these instructions.
 -   Test that your installation was succesful by running `esmvaltool -h`.
 -   If you log into a cluster or other device via `ssh` and your origin machine sends the `locale` environment via the `ssh` connection, make sure the environment is set correctly, specifically `LANG` and `LC_ALL` are set correctly (for GB English UTF-8 encoding these variables must be set to `en_GB.UTF-8`; you can set them by adding `export LANG=en_GB.UTF-8` and `export LC_ALL=en_GB.UTF-8` in your origin or login machines' `.profile`)
 
+## Getting started on MacOS OSX-64
+To install on a MacOS you will have to perform the following installation procedure:
+
+**WARNING**
+The Mac OSX installation is not being continuously tested via CircleCI! If you notice any problems please report them on GitHub.
+
+  - Download and install Anaconda [for OSX](https://repo.anaconda.com/archive/Anaconda3-2019.07-MacOSX-x86_64.sh)(install by e.g. `bash Anaconda3-2019.07-MacOSX-x86_64.sh` or whatever the installer is called);
```

Comment:
I would prefer to link to the distribution page (https://www.anaconda.com/distribution/) instead of the direct link to the package. This is more likely to stay useful long-term and allows the user to see any notes from anaconda.

Also, I personally prefer miniconda (https://docs.conda.io/en/latest/miniconda.html). Functionally equivalent, but a lot fewer packages that we don't need.